web-console/4.0-designs/storage/snapshots/snapshot.md

```diff
@@ -0,0 +1,105 @@
+# PVC/PV Snapshots
+ 
+## Main use cases
```

Comment:
@yuvalgalanti Might be worth putting the main use cases as bullets as "View File" does not render properly.